### PR TITLE
Make edit text single line fixed

### PR DESCRIPTION
**Pull request by arpitagarwal1301**
- Note : maxLines works only with inputType

Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://github.com/DSC-JSS-NOIDA/Plasma-Donor-App/blob/master/CONTRIBUTING.md) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: [Add here what changes were made in this issue and if possible provide links.]

Screenshots for the change:

### CODE REVIEW

**Review by dheerajkotwani — APPROVED** [review]

**Comment by dheerajkotwani** [comment]
@arpitagarwal1301 kindly resolve the conflicts

File: app/src/main/res/layout/donor_form_fragment.xml [hunk 1]
```diff
@@ -72,7 +82,9 @@
                 android:layout_marginStart="30dp"
                 android:layout_marginTop="10dp"
                 android:layout_marginEnd="30dp"
-                android:hint="Mobile" />
+                android:hint="Mobile"
+                android:maxLines="1"
+                android:inputType="text"/>
```
Comment (plazzy99) [inline app/src/main/res/layout/donor_form_fragment.xml:87]: change the input type to ```phone```

**Review by plazzy99 — CHANGES_REQUESTED** [review]
Do the mentioned changes 
The rest all looks good to me.🤞

**Comment by plazzy99** [comment]
also, take a pull from the upstream branch to resolve the merge conflict 
If you have any doubt go through the readme once,
if you still have you can join our slack channel and have your doubts clear.

**Pull request closed without merging**